mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-site_name: "Transit ITS Data Exchange (TIDES) Data Specification Suite"
+site_name: "Transit Integrated Data Exchange (TIDES) Data Specification Suite"
```

Comment:
Would you be okay with: "Transit Integrated Data Exchange Specification (TIDES) Suite"